packages/infra/lib/analytics-platform/analytics-platform-stack.ts

```diff
@@ -69,15 +58,32 @@ function settings(): AnalyticsPlatformsSettings {
       ephemeralStorageSize: cdk.Size.gibibytes(2),
     },
   };
-  const fhirToCsvBatchJob: BatchJobSettings = {
-    imageName: "fhir-to-csv",
-    memory: cdk.Size.mebibytes(1024),
-    cpu: 512,
+  const mergeCsvsLambdaTimeout = Duration.minutes(15).minus(Duration.seconds(10));
+  const mergeCsvs: QueueAndLambdaSettings = {
+    name: "MergeCsvs",
+    entry: "analytics-platform/merge-csvs",
+    lambda: {
+      memory: 4096,
```

Comment:
Ran under 35% for the 1,200 pts, max 800 MB uncompressed files - [link](https://us-west-1.console.aws.amazon.com/cloudwatch/home?region=us-west-1&compassClosed=#metricsV2?graph=~(metrics~(~(~'LambdaInsights~'memory_utilization~'function_name~'tmpCsvMerger~(stat~'Average))~(~'...~(stat~'Maximum)))~region~'us-west-1~view~'timeSeries~legend~(position~'bottom)~stacked~false~title~'memory_utilization*3a*20Average~period~60~id~'widget-1~start~'2025-08-14T15*3a41*3a00.288Z~end~'2025-08-14T21*3a17*3a00.766Z)&query=~'*7bLambdaInsights*2cfunction_name*7d*20function_name*3d*22tmpCsvMerger*22)